runtime/src/main/java/org/corfudb/protocols/CorfuProtocolServerErrors.java

```diff
@@ -0,0 +1,151 @@
+package org.corfudb.protocols;
+
+import com.google.protobuf.ByteString;
+import lombok.extern.slf4j.Slf4j;
+import java.io.ObjectOutputStream;
+
+import org.corfudb.runtime.proto.RpcCommon.UuidMsg;
+import org.corfudb.runtime.proto.ServerErrors.BootstrappedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.DataCorruptionErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.DataOutrankedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.NotBootstrappedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.NotReadyErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.OverwriteErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.ServerErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.TrimmedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.UnknownErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.WrongClusterErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.WrongEpochErrorMsg;
+
+/**
+ * This class provides methods for creating the Protobuf objects defined
+ * in server_errors.proto and are used by several of the service RPCs.
+ */
+@Slf4j
+public class CorfuProtocolServerErrors {
```

Comment:
please consider adding the java doc like @return for these public methods.

Reply:
Done.